duo_client/admin.py

```diff
@@ -2560,6 +2561,7 @@ def update_admin(self, admin_id,
         phone - <str:phone number> (optional)
         password - Deprecated; ignored if specified.
         password_change_required - <bool|None:Whether admin is required to change their password at next login> (optional)
+        status - the status of the adminstrator
```

Comment:
Status is optional - valid values are "Active" and "Disabled".  "Disabled" is not valid for administrators with role = Owner